assets/js/params/Custom/VersionSelect/UpdateFrequencySelect.js

```diff
@@ -69,7 +74,7 @@ export class UpdateFrequencySelect extends React.PureComponent {
         </div>
         {lastCheckDate ? (
           <div className='last-checked'>
-            Checked <time dateTime={this.props.lastCheckDate.toISOString()}>{timeDifference(lastCheckDate, Date.now())}</time>
+            Checked <time dateTime={this.props.lastCheckDate.toISOString()}>{timeDifference(lastCheckDate, Date.now(), i18n)}</time>
```

Comment:
Out of curiosity: how are you tracking which files are translated and which ones aren't? (I was about to suggest a `<Trans>` here but it seems clear that you haven't gotten to this file yet; and of course I want to merge before you do.)

Reply:
At first, we translated a few strings that appear in the workflow page, in order to be able to view our changes easily. After that, we started translating by folder.
Moreover, when changing util functions (such as `timeDifference` here, we of course had to adapt them in every place they are used.